tutorials/optical_tactile_plugin.md

```diff
@@ -192,35 +176,26 @@ simulation, the following topics and services have been advertised:
   `/<namespace>/normal_forces` : Topic where a message is published each
   time the normal forces are computed.
 
-  `/<namespace>/contacts` : Topic where a message is published each time the contacts
-  are computed.
-
 ## Performance
 In order to give an idea about the resources consumed by the plugin, we carried 
 out some simple experiments by setting the parameters to specific values.
 To run the experiments, we've used a Xiaomi Mi Air 13.3 with an Intel Core
 i5-8250U quad-core 1.6GHz.
 
-Mainly, there are two parameters that can affect the computational cost of running
-the plugin: the visualization and the contacts resolutions. The following parameters 
+The main parameter that can affect the computational cost of running
+the plugin is the visualization resolution. The following parameters 
 have been tested for a sensor with a size of 20x20x5 mm:
 
 - `<visualization_resolution>`: A value of 20 allows us to see a sufficient
 number of forces and check what the sensor is looking at. However, if we
-set the value to less than 15, lags can be expected when moving around the
-simulation due to the cost of requesting the `/marker` service too many times.
-
-- `<contacts_resolution>`: Like the previous parameter, a value too low can
-cause the simulation to slow down. The following table aims to provide an idea 
-of how this parameter affects the simulation, with a single instance of the
-plugin loaded in the world:
+set the value to less than 5, performance may decrease.
 
-| `<contacts_resolution>`  | No. of contacts computed | Average RTF (%) |
+| `<visualization_resolution>`  | No. of forces computed (not the same as visualized) | Worst RTF (%) |
```

Comment:
Hmm for the sake of clarity and nicer presentation, I would just put an asterisk in the table, e.g. `No. of forces computed*`, and then a footnote below the table explaining it, e.g. `* blah`.

Isn't the RTF in a pretty small range? What was the range you saw? If it only differs on the scale of 0.01, then just eliminate `Worst`. Right now it sounds like you ran it 10 times and took the worst.

Reply:
In all of the cases, I saw that RTF was around 0.9 and then dropped to those worst values for <1s and then went back to 0.9 or so.

Reply:
In that case, having that explanation for what that column means would be helpful.
Essentially, below the table, there should be explanations for both columns of numbers. That is typical for number reporting to add the English explanation for the table entries so that things are clear.